### ISSUE TITLE
Does it still work?

### ISSUE THREAD

**Issue by soheil-zz** [ben174/git-draw#9]
I tried it the other day but the drawing won't save.

**Comment by Melonbwead** [comment]
nope 😆 , think the layout has been changed on github site for the extension to pick up the right elements on the page.

**Comment by lacymorrow** [comment]
yep, no render button/download button anymore. This project is no longer useful

**Comment by J053Fabi0** [comment]
Same problem, but as you can see on my profile [gitfiti ](https://github.com/gelstudios/gitfiti) is an alternative.

**Comment by Sogl** [comment]
Not working, can't save my draw.

Another project (RU lang README):
https://github.com/sijeko/gerda

**Comment by ochen1** [comment]
Here are my instructions to install an updated, working version. Tested it just now, it's working :smile:

https://github.com/ben174/git-draw/pull/12#issuecomment-737366020
